docs/excel/add-properties-to-basic-cell-values.md

```diff
@@ -0,0 +1,398 @@
+
+- Basic types have a non-error fallback so that calculations can operate on them. For example, consider the formula **=SUM(A1:A3)** where **A1** is **1** (a double with properties), **A2** is **2**, and **A3** is **3**. The sum will return the correct result of **6**. The formula would not work if **A1** was an entity value.
+- When the value of a basic type is used in a calculation, the properties are discarded. In the previous example of **=SUM(A1:A3)** where A1 is a double with properties, the result of **6** will not have any properties.
+- If no icon is specified for a basic type, the cell does not show any icon. If an entity value does not specify an icon, it will still show a default icon in the cell value.
+
+## Formatted number values
+
+You can apply number formatting to values of type **CellValueType.double**. Use the **numberFormat** property in the JSON schema to specify a number format. The following code sample shows the complete schema of a number value formatted as currency. The formatted number value in the code sample displays as **$24.00** in the Excel UI.
+
+```javascript
+// This is an example of the complete JSON of a formatted number value with a property.
+// In this case, the number is formatted as currency.
+async function createCurrencyValue() {
+  await Excel.run(async (context) => {
+    const sheet = context.workbook.worksheets.getActiveWorksheet();
+    const range = sheet.getRange("A1");
+    range.valuesAsJson = [
+      [
+        {
+          type: Excel.CellValueType.double,
+          basicType: Excel.RangeValueType.double,
+          basicValue: 24,
+          numberFormat: "$0.00",
+          properties: {
+            Name: {
+              type: Excel.CellValueType.string,
+              basicValue: "dollar"
+            }
+          }
+        }
+      ]
+    ];
+    await context.sync();
+  });
+}
+```
+
+The number formatting is considered the default format. If the user, or other code, applies formatting to a cell containing a formatted number, the applied format will override the number’s format.
+
+## Card layout
+
+Cell values with properties have a default data type card that the user can view. You can provide a custom card layout to use instead of the default card layout to provide a better user experience for viewing the properties. To do this, add the **layouts** property to the JSON description.
+
+The **layouts** property contains two direct subproperties, **compact** and **card**. The **card** property specifies the appearance of a card when the data type card is open. The compact property is optional, and defines the icon for a value. The icon is shown in the cell value if it is provided. It can also be shown in the card if it is referenced as a sub property. See the [EntityCompactLayoutIcons](/javascript/api/excel/excel.entitycompactlayouticons) enum for a full list of available icons.
```

Comment:
```suggestion
The **layouts** property contains two direct subproperties, **compact** and **card**. The **card** property specifies the appearance of a card when the data type card is open. The compact property is optional and defines the icon for a value. The icon is shown in the cell value if it's provided. It can also be shown in the card if it's referenced as a subproperty. See the [EntityCompactLayoutIcons](/javascript/api/excel/excel.entitycompactlayouticons) enum for a full list of available icons.
```